test/minikube/minikube_base_restore_test.go

```diff
@@ -330,3 +330,47 @@ func TestKubernetesAutoRestore(t *testing.T) {
 		testlib.CheckArchives(t, namespaceName, admin0, opt.DbName, 2, 0)
 	})
 }
+
+func TestKubernetesImportDatabaseSeparateJournal(t *testing.T) {
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+
+	adminOptions := helm.Options{}
+
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, &adminOptions, 1, "")
+
+	admin0 := fmt.Sprintf("%s-nuodb-cluster0-0", helmChartReleaseName)
+
+	verifyPacketFetch(t, namespaceName, admin0)
+
+	t.Run("startDatabaseStatefulSet", func(t *testing.T) {
+		defer testlib.Teardown(testlib.TEARDOWN_DATABASE)
+
+		databaseOptions := &helm.Options{
+			SetValues: map[string]string{
+				"database.autoImport.source":              testlib.IMPORT_ARCHIVE_URL,
```

Comment:
Does this backup have a non-empty journal?

Reply:
Yes it does. Please see https://github.com/nuodb/nuodb-helm-charts/pull/218#issuecomment-859678181.